flowforge/input/Shapes.py

```diff
@@ -0,0 +1,374 @@
+import abc
+import numpy as np
+
+from flowforge.input.UnitConverter import UnitConverter
+
+class Shape:
+    """
+    Abstract base class for defining shapes
+
+    Attributes
+    ----------
+    area : float
+        Area of the given shape
+    perimeter : float
+        Perimeter of the given shape
+    """
+    def __init__(self):
+        pass
+
+    @property
+    @abc.abstractmethod
+    def area(self):
+        raise NotImplementedError
```

Comment:
```suggestion
        pass
```

@abstractmethod already prevents instantiation until a subclass implements it.  Normally you just provide `pass`

Reply:
Fixed